inc/sailag.h

```diff
@@ -141,6 +141,14 @@ typedef enum _sai_lag_attr_t
      */
     SAI_LAG_ATTR_DROP_TAGGED,
 
+    /**
+     * @brief LAG Name
+     *
+     * @type char
+     * @flags IDENTIFY
```

Comment:
what is identify ? please remove this flag, use any of the existing flags

Reply:
Identify flag: 
Attribute with this flag will indicate that this attribute is for the identification
purpose for the client of SAI. SAI implementation doesn't need to handle this attribute at all. 

Please reference the pull request [sonic-sairedis#609](https://github.com/Azure/sonic-sairedis/pull/609).

Reply:
please use standard flags here, you can use create_and_set or create_only